brut.apktool/apktool-lib/src/main/java/brut/androlib/res/AndrolibResources.java

```diff
@@ -1012,9 +1012,8 @@ public void close() throws IOException {
     private boolean mSparseResources = false;
 
     private final static String[] IGNORED_PACKAGES = new String[] {
-            "android", "com.htc", "miui", "com.lge", "com.lge.internal", "yi", "com.miui.core", "flyme",
-            "air.com.adobe.appentry", "FFFFFFFFFFFFFFFFFFFFFF" };
+            "android", "com.htc", "com.lge", "com.lge.internal", "yi", "flyme", "air.com.adobe.appentry",
+            "FFFFFFFFFFFFFFFFFFFFFF" };
 
-    private final static String[] ALLOWED_PACKAGES = new String[] {
-            "com.miui" };
+    private final static String[] ALLOWED_PACKAGES = new String[] { };
```

Comment:
If the reason I added `ALLOWED_PACKAGES` is no longer needed (ie miui). We can cleanup a lot of the codebase as this was a hacky fix I had to do for MIUI back when we actually supported it.

Reply:
Probably. What I do know is that com.miui package doesn't even exist.
Nowadays the packages are:
miui/com.miui.core
android.miui/com.miui.rom
miui.system/com.miui.system

Reply:
Thanks, I'll clean it up and remove the workaround. Would help to cut down the codebase.